### ISSUE TITLE
Jail naming, start/stop/destroy issue

### ISSUE THREAD

**Issue by OnigamiSama** [BastilleBSD/bastille#87]
Hello !

I created a new jail "sonarr-devel" to test the new version of this PVR.

I wanted to stop the "classic" sonarr jail but had this behavior :

```
zutsuu% sudo bastille stop sonarr
Password:
[: sonarr: unexpected operator
[: sonarr: unexpected operator

[sonarr-devel]:
sonarr-devel: removed
zutsuu% sudo bastille list
 JID             IP Address      Hostname                      Path
 jackett         192.168.1.112   jackett                       /usr/local/bastille/jails/jackett/root
 lidarr          192.168.1.115   lidarr                        /usr/local/bastille/jails/lidarr/root
 minecraft       192.168.1.120   minecraft                     /usr/local/bastille/jails/minecraft/root
 radarr          192.168.1.114   radarr                        /usr/local/bastille/jails/radarr/root
 sonarr          192.168.1.113   sonarr                        /usr/local/bastille/jails/sonarr/root
 transmission    192.168.1.111   transmission                  /usr/local/bastille/jails/transmission/root
```
Here I tried to stop the sonarr jail but instead the sonarr-devel jail got shut down

```
zutsuu% sudo bastille stop sonarr
[sonarr]:
sonarr: removed
```
Running bastille stop a second time shut down the sonarr jail, the sonarr-devel is already shut down

```
zutsuu% sudo bastille start sonarr-devel
[sonarr-devel]:
sonarr-devel: created
```
Starting seems to work fine tho

```
zutsuu% sudo bastille destroy sonarr
Jail running.
See 'bastille stop sonarr'.
```
Can't destroy the sonarr jail because sonarr-devel is running

**Comment by OnigamiSama** [comment]
Also I can stop sonarr-devel even when I've already deleted the sonarr jail

```
zutsuu% sudo bastille stop sonarr
[sonarr-devel]:
sonarr-devel: removed
```

**Comment by JRGTH** [comment]
Hi, the issue here is dealing with names containing dash/hyphen causing grep to pass unwanted container names with such characters.

However this issue is fixed in [this commit](https://github.com/BastilleBSD/bastille/pull/88/commits/0c78ebae8865778e2bf802d23e063865c9c6bb21) using awk instead, now names containing multiple dash/hyphen will be respected.

As for the "unexpected operator", it is also fixed by adding the missing quotes.

Regards!

**Comment by cedwards** [comment]
@JRGTH your patch covered start/stop. Does `destroy` also need to be updated?

**Comment by JRGTH** [comment]
> @JRGTH your patch covered start/stop. Does `destroy` also need to be updated?

Hi cedwards, the above destroy issue was also caused by the name handling issue, destroy command should work ok with multiple dash since the stop command now will stop them accordingly:
```
root@server: ~# bastille destroy test--dev
Deleting Jail: test--dev.
Note: jail console logs archived.
/mnt/storage/bastille/logs/test--dev_console.log-2019-12-27

root@server: ~#
```

Regards!

**Comment by cedwards** [comment]
Is this now resolved in develop? Can the issue be closed?

**Comment by JRGTH** [comment]
> Is this now resolved in develop? Can the issue be closed?

Hi, yes this issues has been resolved, and the [fix #89](https://github.com/BastilleBSD/bastille/pull/89) also covered the create and destroy commands as well.

Regards
